v2/src/ctrlfbe/basedata.py

```diff
@@ -21,29 +27,61 @@ def build_data(self):
         }
         return model_data, issue_data
 
+    def build_update_data(self):
+        issue_data = {
+            "title": self.request.data["new_title"],
+            "reason": self.request.data["reason"],
+            "status": CtrlfIssueStatus.REQUESTED,
+            "action": CtrlfActionType.UPDATE,
+        }
+        return issue_data
```

Comment:
메소드명과 내용이 완벽히 일치안되는 것 같은데, 일치시켜주면 어떨까?? 

```suggestion
    def build_issue_data_for_update(self):
        return {
            "title": self.request.data["new_title"],
            "reason": self.request.data["reason"],
            "status": CtrlfIssueStatus.REQUESTED,
            "action": CtrlfActionType.UPDATE,
        }
```